src/initialise.js

```diff
@@ -93,31 +95,23 @@ module.exports = async app => {
 	 */
 	const installation = await app.auth(installationId).catch(err => {
 		throw new InitialisationError(
-			`Failed to authenticate as installation ${installationId}`,
-			{ err }
+			`Failed to authenticate as installation ${installationId}`, {
+				err
+			}
 		);
 	});
 
 	logger.debug(`Authenticated as installation ${installationId}`);
 
 	try {
-		// Get the repositories this app is installed on.
-		// @see https://octokit.github.io/rest.js/#api-Apps-getInstallationRepositories
-		const repositories = await installation.paginate(
-			installation.apps.getInstallationRepositories({ per_page: 100 }),
-			res => res.data.repositories // Pull out only the list of repositories from each response.
-		);
-
-		// Save each repository to our global map of repositories.
-		repositories.forEach(({ id, name }) => {
-			repositoryStore.set(id, { id, name });
-		});
-	} catch (err) {
+		await repositories.refresh(installation);
+		return installation;
+	}
+	catch (err) {
 		throw new InitialisationError(
-			"Failed to fetch repository information with apps.getInstallationRepositories",
-			{ err }
+			`Failed to refresh the repository store from installation ${installationId}`, {
+				err
+			}
 		);
 	}
-
-	logger.info(`Loaded ${repositoryStore.size} repositories`);
```

Comment:
This log message seems useful - perhaps it could be resurrected in the new `repositories.refresh()` method?

Reply:
Good call — will do.